From: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Date: Fri, 7 Aug 2015 17:07:52 -0400
Subject: 9004-libvirtd-group-name

---
 daemon/libvirtd.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/daemon/libvirtd.conf b/daemon/libvirtd.conf
index 5f5a0d2..bad44a5 100644
--- a/daemon/libvirtd.conf
+++ b/daemon/libvirtd.conf
@@ -87,7 +87,7 @@
 # without becoming root.
 #
 # This is restricted to 'root' by default.
-unix_sock_group = "libvirt"
+unix_sock_group = "libvirtd"
 
 # Set the UNIX socket permissions for the R/O socket. This is used
 # for monitoring VM status only
