From: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Date: Wed, 5 Aug 2015 02:00:55 -0400
Subject: 9021-fix-uint64_t

===================================================================
---
 src/lxc/lxc_container.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/lxc/lxc_container.c b/src/lxc/lxc_container.c
index 7154804..3137dc9 100644
--- a/src/lxc/lxc_container.c
+++ b/src/lxc/lxc_container.c
@@ -42,6 +42,7 @@
 #include <linux/unistd.h>
 
 /* For MS_MOVE */
+#include <stdint.h>
 #include <linux/fs.h>
 
 #if WITH_CAPNG
