From: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Date: Fri, 7 Aug 2015 17:07:52 -0400
Subject: 9021-fix-uint64_t

===================================================================
---
 src/lxc/lxc_container.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/lxc/lxc_container.c b/src/lxc/lxc_container.c
index 9a9ae5c..20fbdb3 100644
--- a/src/lxc/lxc_container.c
+++ b/src/lxc/lxc_container.c
@@ -43,6 +43,7 @@
 #include <linux/unistd.h>
 
 /* For MS_MOVE */
+#include <stdint.h>
 #include <linux/fs.h>
 
 #if WITH_CAPNG
