From: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Date: Fri, 7 Aug 2015 17:07:52 -0400
Subject: 9031-enable-lxc-apparmor

---
 src/lxc/lxc.conf | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/lxc/lxc.conf b/src/lxc/lxc.conf
index 8df4601..582d62d 100644
--- a/src/lxc/lxc.conf
+++ b/src/lxc/lxc.conf
@@ -20,11 +20,13 @@
 # to 'none' instead.
 #
 #security_driver = "selinux"
+security_driver = "apparmor"
 
 # If set to non-zero, then the default security labeling
 # will make guests confined. If set to zero, then guests
 # will be unconfined by default. Defaults to 0.
 #security_default_confined = 1
+security_default_confined = 1
 
 # If set to non-zero, then attempts to create unconfined
 # guests will be blocked. Defaults to 0.
