From: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Date: Fri, 7 Aug 2015 17:07:53 -0400
Subject: 9032-lxc-allow-no-security-driver

===================================================================
---
 src/lxc/lxc_driver.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/src/lxc/lxc_driver.c b/src/lxc/lxc_driver.c
index 31fb470..487dd9b 100644
--- a/src/lxc/lxc_driver.c
+++ b/src/lxc/lxc_driver.c
@@ -1560,6 +1560,11 @@ lxcSecurityInit(virLXCDriverConfigPtr cfg)
                                                       cfg->securityDefaultConfined,
                                                       cfg->securityRequireConfined);
     if (!mgr)
+        mgr = virSecurityManagerNew(NULL, LXC_DRIVER_NAME, false,
+                                                      cfg->securityDefaultConfined,
+                                                      cfg->securityRequireConfined);
+
+    if (!mgr)
         goto error;
 
     return mgr;
