Index: libvirt-1.2.1/src/conf/cpu_conf.c
===================================================================
--- libvirt-1.2.1.orig/src/conf/cpu_conf.c	2013-12-03 20:44:44.000000000 +0400
+++ libvirt-1.2.1/src/conf/cpu_conf.c	2014-01-17 15:33:37.493590025 +0400
@@ -361,7 +361,8 @@
         goto error;
 
     if (n > 0) {
-        if (!def->model && def->mode != VIR_CPU_MODE_HOST_MODEL) {
+        if (!def->model && def->mode != VIR_CPU_MODE_HOST_MODEL
+            && def->mode != VIR_CPU_MODE_HOST_PASSTHROUGH) {
             virReportError(VIR_ERR_XML_ERROR, "%s",
                            _("Non-empty feature list specified without "
                              "CPU model"));
@@ -576,6 +577,7 @@
 
     if (!def->model &&
         def->mode != VIR_CPU_MODE_HOST_MODEL &&
+        def->mode != VIR_CPU_MODE_HOST_PASSTHROUGH &&
         def->nfeatures) {
         virReportError(VIR_ERR_INTERNAL_ERROR, "%s",
                        _("Non-empty feature list specified without CPU model"));
