From: Guido Guenther <agx@sigxcpu.org>
Date: Fri, 7 Mar 2008 09:08:26 +0100
Subject: Don't enable default network on boot

to not interfere with existing network configurations
---
 src/Makefile.am | 3 ---
 src/Makefile.in | 3 ---
 2 files changed, 6 deletions(-)

Index: libvirt-1.1.4/src/Makefile.am
===================================================================
--- libvirt-1.1.4.orig/src/Makefile.am	2013-11-12 11:48:30.027867074 -0600
+++ libvirt-1.1.4/src/Makefile.am	2013-11-12 11:48:30.003867074 -0600
@@ -2545,9 +2545,6 @@
 	    cp $(DESTDIR)$(confdir)/qemu/networks/default.xml.t \
 	      $(DESTDIR)$(confdir)/qemu/networks/default.xml && \
 	    rm $(DESTDIR)$(confdir)/qemu/networks/default.xml.t; }
-	test -e $(DESTDIR)$(confdir)/qemu/networks/autostart/default.xml || \
-	   ln -s ../default.xml \
-	    $(DESTDIR)$(confdir)/qemu/networks/autostart/default.xml
 endif WITH_NETWORK
 
 uninstall-local:: uninstall-init uninstall-systemd
Index: libvirt-1.1.4/src/Makefile.in
===================================================================
--- libvirt-1.1.4.orig/src/Makefile.in	2013-11-12 11:48:30.027867074 -0600
+++ libvirt-1.1.4/src/Makefile.in	2013-11-12 11:48:30.015867074 -0600
@@ -9594,9 +9594,6 @@
 @WITH_NETWORK_TRUE@	    cp $(DESTDIR)$(confdir)/qemu/networks/default.xml.t \
 @WITH_NETWORK_TRUE@	      $(DESTDIR)$(confdir)/qemu/networks/default.xml && \
 @WITH_NETWORK_TRUE@	    rm $(DESTDIR)$(confdir)/qemu/networks/default.xml.t; }
-@WITH_NETWORK_TRUE@	test -e $(DESTDIR)$(confdir)/qemu/networks/autostart/default.xml || \
-@WITH_NETWORK_TRUE@	   ln -s ../default.xml \
-@WITH_NETWORK_TRUE@	    $(DESTDIR)$(confdir)/qemu/networks/autostart/default.xml
 
 uninstall-local:: uninstall-init uninstall-systemd
 @WITH_LIBVIRTD_TRUE@	rmdir "$(DESTDIR)$(localstatedir)/lib/libvirt/lockd/files" ||:
