From: =?utf-8?q?Guido_G=C3=BCnther?= <agx@sigxcpu.org>
Date: Sat, 14 Jan 2012 18:09:01 +0100
Subject: Don't fail if we can't setup avahi

---
 src/rpc/virnetserver.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

Index: libvirt-1.1.4/src/rpc/virnetserver.c
===================================================================
--- libvirt-1.1.4.orig/src/rpc/virnetserver.c	2013-11-12 11:48:33.219867046 -0600
+++ libvirt-1.1.4/src/rpc/virnetserver.c	2013-11-12 11:48:33.207867046 -0600
@@ -1070,8 +1070,7 @@
 
     virObjectLock(srv);
 
-    if (srv->mdns &&
-        virNetServerMDNSStart(srv->mdns) < 0)
+    if (srv->mdns)
         goto cleanup;
 
     srv->quit = false;
