Index: libvirt-1.1.4/src/qemu/qemu_capabilities.c
===================================================================
--- libvirt-1.1.4.orig/src/qemu/qemu_capabilities.c	2013-11-12 11:48:36.215867019 -0600
+++ libvirt-1.1.4/src/qemu/qemu_capabilities.c	2013-11-12 11:48:36.203867019 -0600
@@ -705,6 +705,7 @@
     if (virQEMUCapsIsValidForKVM(hostarch, guestarch)) {
         const char *const kvmbins[] = { "/usr/libexec/qemu-kvm", /* RHEL */
                                         "qemu-kvm", /* Fedora */
+										"kvm-spice", /* qemu-kvm-spice Ubuntu package */
                                         "kvm" }; /* Upstream .spec */
 
         for (i = 0; i < ARRAY_CARDINALITY(kvmbins); ++i) {
