From: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Date: Wed, 5 Aug 2015 02:00:56 -0400
Subject: Support-incoming-migration-from-13.10-hosts

===================================================================
---
 src/qemu/qemu_migration.c | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/src/qemu/qemu_migration.c b/src/qemu/qemu_migration.c
index 5eec741..8a21b5a 100644
--- a/src/qemu/qemu_migration.c
+++ b/src/qemu/qemu_migration.c
@@ -2109,6 +2109,13 @@ static char
                         VIR_STRDUP(def->os.machine, "pc-1.0-precise") < 0)
                     goto cleanup;
             }
+            if (STREQ_NULLABLE(vm->def->os.machine, "pc-i440fx-1.5")) {
+                VIR_FREE(vm->def->os.machine);
+                VIR_FREE(def->os.machine);
+                if (VIR_STRDUP(vm->def->os.machine, "pc-i440fx-1.5-saucy") < 0 ||
+                        VIR_STRDUP(def->os.machine, "pc-i440fx-1.5-saucy") < 0)
+                    goto cleanup;
+            }
         }
 
         rv = qemuDomainDefFormatLive(driver, def, false, true);
@@ -2447,6 +2454,11 @@ qemuMigrationPrepareAny(virQEMUDriverPtr driver,
                 if (VIR_STRDUP(vm->def->os.machine, "pc-1.0-precise") < 0)
                     goto endjob;
             }
+            if (STREQ_NULLABLE(vm->def->os.machine, "pc-i440fx-1.5")) {
+                VIR_FREE(vm->def->os.machine);
+                if (VIR_STRDUP(vm->def->os.machine, "pc-i440fx-1.5-saucy") < 0)
+                    goto endjob;
+            }
     }
  
     /* Start the QEMU daemon, with the same command-line arguments plus
