From: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Date: Fri, 7 Aug 2015 17:07:53 -0400
Subject: ubuntu_machine_type

---
 src/qemu/qemu_domain.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/qemu/qemu_domain.c b/src/qemu/qemu_domain.c
index db8554b..4767ef2 100644
--- a/src/qemu/qemu_domain.c
+++ b/src/qemu/qemu_domain.c
@@ -992,6 +992,7 @@ qemuDomainDefPostParse(virDomainDefPtr def,
             !STRPREFIX(def->os.machine, "pc-1.") &&
             !STRPREFIX(def->os.machine, "pc-i440") &&
             !STREQ(def->os.machine, "pc") &&
+			!STREQ(def->os.machine, "ubuntu") &&
             !STRPREFIX(def->os.machine, "rhel"))
             break;
         addPCIRoot = true;
@@ -3180,5 +3181,6 @@ qemuDomainMachineIsI440FX(const virDomainDef *def)
             STRPREFIX(def->os.machine, "pc-0.") ||
             STRPREFIX(def->os.machine, "pc-1.") ||
             STRPREFIX(def->os.machine, "pc-i440") ||
+			STREQ(def->os.machine, "ubuntu") ||
             STRPREFIX(def->os.machine, "rhel"));
 }
